src/meson.build

```diff
@@ -2,20 +2,22 @@ kiwix_sources = [
   'library.cpp',
   'manager.cpp',
   'reader.cpp',
-  'searcher.cpp',
```

Comment:
searcher.cpp is using xapian only to search on external fulltext index.
If the fulltext is embedded in the zim file, it use libzim to do the search (and not use xapian directly).

It would be better fix searcher.cpp than simply not compiling it. Especially if we install the searcher.h header all the time (and we do).

Reply:
This fix is a bit more complex than just fixing the `meson.build` file. You may not expected to do such changes. If you don't want to do this change yourself, open a issue and I will do it myself.

Of course, if you want to do it, it would be welcome :)